packages/estatico-boilerplate/src/assets/js/helpers/app.js

```diff
@@ -75,25 +75,24 @@ class App {
     return window[namespace].modules[moduleName];
   }
 
-  isInitialised($element, moduleName) {
+  isInitialised(element, moduleName) {
     // jQuery 3 does not allow kebab-case in data() when retrieving whole data object https://jquery.com/upgrade-guide/3.0/#breaking-change-data-names-containing-dashes
```

Comment:
This line is unnecessary when removing jQuery :)

Reply:
Very good point, removed